modules/rpc/handler.go

```diff
@@ -26,6 +26,7 @@ import (
 	"go.uber.org/fx"
 	"go.uber.org/fx/internal/fxcontext"
 	"go.uber.org/fx/service"
+	"go.uber.org/fx/uauth"
```

Comment:
I am keeping it as is. Let's get an issue open to discuss naming. `u` in `uauth`, `uhttp` or `ulog` comes from `uberFx`.  We would need a generic naming overhaul